sdk/src/Services/DynamoDBv2/Custom/DocumentModel/_async/DocumentBatchGet.Async.cs

```diff
@@ -22,6 +22,8 @@
 using Amazon.DynamoDBv2.Model;
 using Amazon.Runtime;
 using Amazon.Runtime.Internal;
+using Amazon.Runtime.Telemetry.Tracing;
+using AWSSDK.DynamoDBv2;
```

Comment:
I don't understand where this namespace is coming from. There shouldn't be a namespace called "AWSSDK.DynamoDBv2"

Reply:
I had added the new internal static class `DynamoDBTelemetry` in the `AWSSDK.DynamoDBv2` namespace. I have changed the namespace for that class to `Amazon.DynamoDBv2` which is an existing one.